lib/modules/manager/argocd/extract.ts

```diff
@@ -66,28 +84,27 @@ function createDependency(
   };
 }
 
-export function extractPackageFile(
-  content: string,
-  fileName: string,
-  _config?: ExtractConfig
-): PackageFileContent | null {
-  // check for argo reference. API version for the kind attribute is used
-  if (fileTestRegex.test(content) === false) {
-    return null;
+export function processAppSpec(
```

Comment:
I'm still not happy to export this, can't you rewrite the tests to use extract function only?